packages/cli/tests/unit/record/testCasesComplete.spec.ts

```diff
@@ -1,52 +1,75 @@
-import sinon from 'sinon';
```

Comment:
But barely, and it doesn't really bring anything over what jest already provides. I figured it might be a good idea to incrementally remove it while I'm touching something anyway. I can rewrite it with sinon if you'd like, but I figure the less dependencies the better :)